apps/service-portal/src/components/GlobalModules/GlobalModules.tsx

```diff
@@ -0,0 +1,80 @@
+import {
```

Comment:
is this file intended to be part of this PR? how does service-portal and api-catalogue connect?

Reply:
I did a `yarn format` to format my changes, admit I reviewed the staged file in a hurry. There is no connection and should no be a part of this PR. I will revert and fix this. Well spotted!